### DISCUSSION TITLE
What json setting affects window pane border color?

### DISCUSSION THREAD

**Discussion by jtmoon79** [microsoft/terminal#13468] [Q&A] [answered]
I want to change the window pane border color.  That is, the border between different shell instances within the same Windows Terminal tab/window. What json setting can control that?

I'm most interested in the unfocused window pane border color. But if you also know the focused window pane border color then please share.

**Comment by zadjii-msft** [comment, marked as answer]
This isn't supported quite yet. That's tracked in #3061 (and to a lesser degree, #8564). 

Currently, the Terminal just defaults to your Accent color to highlight the active pane.
